### DISCUSSION TITLE
Make commands for donators (like in the old Kohl's admin)

### DISCUSSION THREAD

**Discussion by TheHackerLxr** [kohls-admin/kohls-admin#23] [Ideas]
Please add the commands fire, cape, particles, sparkles, BIGhead, SMALLhead, normalhead, and also make them as regular commands, not only for donators but also for admins

**Comment by kohltastrophe** [comment]
Fun commands will be coming soon!

**Reply by TheHackerLxr** [reply]
Also, please add the commands: hat (puts on any accessory), shirt, pants, t-shirt. Make these commands for donors and administrators. And also I have not seen yet in any admin panel that you can put on layered clothing that Roblox recently made, for them there will be commands for example 3dshirt, 3dpants. Thanks for adding!
